css/style.css

```diff
@@ -1309,9 +1333,20 @@ body#Home #content > .intro .download
 
 body#Home #content > .intro .download .btn
 {
-    margin-bottom: 0.3em;
+    margin: 0 0.3em 0.3em 0.3em;
+}
+
+body#Home #content > .intro .download .btn.action
+{
+    border-color: #98312A;
 }
 
+body#Home #content > .intro .download .btn.action:hover
+{
+    border-color: #943029;
+}
+
+
```

Comment:
FYI I think a couple of PRs ago we agreed that inlining page-specific CSS into the page makes sense, because (a) it helps to cleanup the massive `style.css` and (b) it's also faster for clients and (c) the `style.css` is now used at many different places (main page, docs, forum, code.dlang.org, tour.dlang.org, ...) s.t. every addition adds to size that needs to be downloaded by all those pages.